libbeat/common/uuid.go

```diff
@@ -0,0 +1,22 @@
+package common
+
+import (
+	"crypto/rand"
+	"fmt"
+)
+
+// GenerateUUID is used to generate a random UUID
+// This was copied from https://github.com/hashicorp/uuid/blob/master/uuid.go
```

Comment:
@tsg I think that is more a discussion about which version of UUID we want to use. Host and time sounds like version 1 UUID. I prefer version 4 as it is fully random and does not have any dependencies on time or host. I would not use the UUID to try to figure out host and time. If this information is needed, I would send it separately.